src/app/games/gos/go/GoRules.ts

```diff
@@ -0,0 +1,96 @@
+import { MGPOptional } from '@everyboard/lib';
+import { NumberConfig, RulesConfigDescription, RulesConfigDescriptionLocalizable } from 'src/app/components/wrapper-components/rules-configuration/RulesConfigDescription';
+import { AbstractGoRules } from '../AbstractGoRules';
+import { GoState } from '../GoState';
+import { GoPiece } from '../GoPiece';
+import { GobanUtils } from 'src/app/jscaip/GobanUtils';
+import { Coord } from 'src/app/jscaip/Coord';
+import { PlayerNumberMap } from 'src/app/jscaip/PlayerMap';
+import { GoPhase } from '../GoPhase';
+import { MGPValidators } from 'src/app/utils/MGPValidator';
+import { GobanConfig } from 'src/app/jscaip/GobanConfig';
+import { Direction } from 'src/app/jscaip/Direction';
+import { Orthogonal } from 'src/app/jscaip/Orthogonal';
+import { OrthogonalGoGroupDatasFactory } from '../GoGroupDatasFactory';
+import { GroupDatasFactory } from 'src/app/jscaip/BoardDatas';
+
+export type GoConfig = GobanConfig & {
+
+    handicap: number;
+};
+
+export class GoRules extends AbstractGoRules<GoConfig> {
+
+    private static singleton: MGPOptional<GoRules> = MGPOptional.empty();
+
+    public static readonly RULES_CONFIG_DESCRIPTION: RulesConfigDescription<GoConfig> =
+        new RulesConfigDescription<GoConfig>({
+            name: (): string => $localize`19 x 19`,
```

Comment:
proposition: utiliser un signe mathématique × plutôt qu'une lettre x

Reply:
comme ?

Reply:
19 x 19 vs. 19 × 19

Reply:
j'veux bien mais 1. personne d'autre que toi verra ça
2. c'est inécrivable et donc ingreppable, c'est déjà pénible avec l'espace insécable si on peux éviter les point virgules grecs aussi ça m'arrangerais x)

Reply:
J'accepterais bien * mais meh

Reply:
1. Si, tous les autistes de typographie verront l'erreur de la honte !
2. Suffit d'appuyer sur la touche × pourtant !
3. Allez, TPR